docs/development/extensions-core/avro.md

```diff
@@ -22,47 +22,43 @@ title: "Apache Avro"
-two Avro Parsers for stream ingestion and Hadoop batch ingestion. 
-See [Avro Hadoop Parser](../../ingestion/data-formats.md#avro-hadoop-parser) and [Avro Stream Parser](../../ingestion/data-formats.md#avro-stream-parser)
-for more details about how to use these in an ingestion spec.
+The [Avro Stream Parser](../../ingestion/data-formats.md#avro-stream-parser) is deprecated.
 
-Additionally, it provides an InputFormat for reading Avro OCF files when using
-[native batch indexing](../../ingestion/native-batch.md), see [Avro OCF](../../ingestion/data-formats.md#avro-ocf)
-for details on how to ingest OCF files.
+## Load the Avro extension
 
-Make sure to [include](../../development/extensions.md#loading-extensions) `druid-avro-extensions` in the extensions load list.
+To use the Avro extension, add the `druid-avro-extensions` to the list of loaded extensions. See [Loading extensions](../../development/extensions.md#loading-extensions).
 
-### Avro Types
+## Avro Types
 
-Druid supports most Avro types natively, there are however some exceptions which are detailed here.
+Druid supports most Avro types natively. This section describes some  exceptions.
 
-#### Unions
+### Unions
 Druid has two modes for supporting `union` types.
 
-The default mode will treat unions as a single value regardless of the type it is populated with.
+The default mode treats unions as a single value regardless of the type of data populating the union.
 
-If you wish to operate on each different member of a union however you can set `extractUnionsByType` on the Avro parser in which case unions will be expanded into nested objects according to the following rules:
-* Primitive types and unnamed complex types are keyed their type name. i.e `int`, `string`
-* Complex named types are keyed by their names, this includes `record`, `fixed` and `enum`.
-* The Avro null type is elided as its value can only ever be null
+If you want to operate on individual members of a union, set `extractUnionsByType` on the Avro parser. This configuration expands union values into nested objects according to the following rules:
+- Primitive types and unnamed complex types are keyed their type name. For example: `int`, `string`
+- Complex named types are keyed by their names, this includes `record`, `fixed` and `enum`.
+- The Avro null type is elided as its value can only ever be null
 
-This is safe because an Avro union can only contain a single member of each unnamed type and duplicates of the same named type are not allowed.
-i.e only a single array is allowed, multiple records (or other named types) are allowed as long as each has a unique name.
+This is safe because an Avro union can only contain a single member of each unnamed type and duplicates of the same named type are not allowed. For example: only a single array is allowed, multiple records (or other named types) are allowed as long as each has a unique name.
```

Comment:
```suggestion
This is safe because an Avro union can only contain a single member of each unnamed type and duplicates of the same named type are not allowed. For example, only a single array is allowed, multiple records (or other named types) are allowed as long as each has a unique name.
```